crates/oxc_regular_expression/src/display.rs

```diff
@@ -0,0 +1,565 @@
+            format!(r"\{octal}")
+        }
+        CharacterKind::SingleEscape => match ch {
+            '\n' => String::from(r"\n"),
+            '\r' => String::from(r"\r"),
+            '\t' => String::from(r"\t"),
+            '\u{b}' => String::from(r"\v"),
+            '\u{c}' => String::from(r"\f"),
+            '\u{8}' => String::from(r"\b"),
+            '\u{2D}' => String::from(r"\-"),
+            _ => format!(r"\{ch}"),
+        },
+    };
+
+    (result, false)
+}
+
+#[cfg(test)]
+mod test {
+    use oxc_allocator::Allocator;
+
+    type Case<'a> = (
+        &'a str,
+        /* expected display, None means expect the same as original */ Option<&'a str>,
+    );
+
+    static CASES: &[Case] = &[
+        ("/ab/", None),
+        ("/ab/u", None),
+        ("/abc/i", None),
+        ("/abc/iu", None),
+        ("/a*?/i", None),
+        ("/a*?/iu", None),
+        ("/emo👈🏻ji/", None),
+        ("/emo👈🏻ji/u", None),
+        ("/ab|c/i", None),
+        ("/ab|c/iu", None),
+        ("/a|b+|c/i", None),
+        ("/a|b+|c/iu", None),
+        ("/(?=a)|(?<=b)|(?!c)|(?<!d)/i", None),
+        ("/(?=a)|(?<=b)|(?!c)|(?<!d)/iu", None),
+        (r"/(cg)(?<n>cg)(?:g)/", None),
+        (r"/(cg)(?<n>cg)(?:g)/u", None),
+        (r"/^(?=ab)\b(?!cd)(?<=ef)\B(?<!gh)$/", None),
+        (r"/^(?=ab)\b(?!cd)(?<=ef)\B(?<!gh)$/u", None),
+        (r"/^(?<!ab)$/", None),
+        (r"/^(?<!ab)$/u", None),
+        (r"/[abc]/", None),
+        (r"/[abc]/u", None),
+        (r"/[a&&b]/v", None),
+        (r"/[a&&b]/vu", None),
+        (r"/[a--b]/v", None),
+        (r"/[a--b]/vu", None),
+        (r"/[^a--b--c]/v", None),
+        (r"/[^a--b--c]/vu", None),
+        (r"/[a[b[c[d[e[f[g[h[i[j[k[l]]]]]]]]]]]]/v", None),
+        (r"/[a[b[c[d[e[f[g[h[i[j[k[l]]]]]]]]]]]]/vu", None),
+        (r"/[\q{abc|d|e|}]/v", None),
+        (r"/[\q{abc|d|e|}]/vu", None),
+        (r"/\p{Basic_Emoji}/v", None),
+        (r"/\p{Basic_Emoji}/vu", None),
+        (r"/[|\]]/", None),
+        (r"/[|\]]/u", None),
+        (r"/c\]/", None),
+        (r"/c\]/u", None),
+        ("/a{0}|b{1,2}|c{3,}/i", None),
+        ("/a{0}|b{1,2}|c{3,}/iu", None),
+        (r"/Em🥹j/", None),
+        (r"/Em🥹j/u", None),
+        (r"/\n\cM\0\x41\./", None),
+        (r"/\n\cM\0\x41\./u", None),
+        (r"/\n\cM\0\x41\u1234\./", None),
+        (r"/\n\cM\0\x41\u1234\./u", None),
+        (r"/[\bb]/", None),
+        (r"/[\bb]/u", None),
+        (r"/\d+/g", None),
+        (r"/\d+/gu", None),
+        (r"/\D/g", None),
+        (r"/\D/gu", None),
+        (r"/\w/g", None),
+        (r"/\w/gu", None),
+        (r"/\w+/g", None),
+        (r"/\w+/gu", None),
+        (r"/\s/g", None),
+        (r"/\s/gu", None),
+        (r"/\s+/g", None),
+        (r"/\s+/gu", None),
+        (r"/\t\n\v\f\r/", None),
+        (r"/\t\n\v\f\r/u", None),
+        // we lose the flags ordering
+        ("/abcd/igv", Some("/abcd/giv")),
+        ("/abcd/igvu", Some("/abcd/givu")),
```

Comment:
`u` and `v` are not allowed to live together.
```suggestion
        ("/abcd/igu", Some("/abcd/giv")),
```

Reply:
Yes, I changed them in bulk via a macro😅 Fixing it right now.